docs/book/src/multiversion-tutorial/testdata/project/cmd/main.go

```diff
@@ -64,8 +64,6 @@ func init() {
 /*
  */
 func main() {
-	/*
-	 */
```

Comment:
Are you sure that is redundant and is not been used to render the code properly in the docs?
Did you check how this part is used in the docs?